sw/device/lib/dif/dif_alert_handler.c

```diff
@@ -23,11 +23,10 @@ dif_alert_handler_result_t dif_alert_handler_init(
     return kDifAlertHandlerBadArg;
   }
 
-  // TODO: We currently don't support more than 16 alerts correctly.
-  // See: #3826
-  if (params.alert_count > 16) {
-    return kDifAlertHandlerBadArg;
-  }
+  // TODO(timothytrippel): should we be checking:
```

Comment:
Can this be a bug number?

Reply:
It doesn't seem anything should go wrong if 0 alerts are configured. However, I added a check for the case when the params.alert_count is > what is supported by the hardware (since this is a generated macro).